mobile/src/styles/TextStyles.ts

```diff
@@ -18,14 +18,19 @@ export const TextStyles = StyleSheet.create({
   },
 
   body1: {
-    fontSize: 18,
+    fontSize: 16,
     fontFamily: 'Gotham-Light',
     fontWeight: 'normal',
   },
   body2: {
     fontSize: 12,
     fontFamily: 'Gotham-Light',
   },
+  body3: {
+    fontSize: 18,
+    fontFamily: 'Gotham-Light',
+    fontWeight: 'normal',
+  },
```

Comment:
Definitely seems weird to have body3 be *larger* than body1 and 2, can you move this into ChatScreen? I don't think we're going to use this anywhere else